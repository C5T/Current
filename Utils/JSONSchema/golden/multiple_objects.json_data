{"s":"string","b":true}
{"s":"another string","b":false}
{"s":"duh","b":true,"optional":[{"string":"s","maybe":true},{"string":"s2"}]}
