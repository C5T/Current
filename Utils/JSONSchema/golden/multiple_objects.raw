{"Object":{"fields":{"b":{"Bool":{"values_false":1,"values_true":2,"nulls":0}},"optional":{"Array":{"element":{"Object":{"fields":{"maybe":{"Bool":{"values_false":0,"values_true":1,"nulls":1}},"string":{"String":{"values":{"s":1,"s2":1},"instances":2,"nulls":0}}},"instances":2,"nulls":0}},"instances":1,"nulls":1}},"s":{"String":{"values":{"another string":1,"duh":1,"string":1},"instances":3,"nulls":0}}},"instances":3,"nulls":0}}