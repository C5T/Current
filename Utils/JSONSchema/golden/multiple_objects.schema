// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_optional_Element_Object) {
  CURRENT_FIELD(string, std::string);
  CURRENT_FIELD(maybe, bool);
};

CURRENT_STRUCT(Schema_Object) {
  CURRENT_FIELD(s, std::string);
  CURRENT_FIELD(b, bool);
  CURRENT_FIELD(optional, std::vector<Schema_optional_Element_Object>);
};

using Schema = Schema_Object;
