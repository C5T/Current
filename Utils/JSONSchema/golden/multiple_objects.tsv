Field	Type	Set	Unset/Null	Values	Details
Schema	Object	3	0	3 fields	s, b, optional
Schema.s	String	3	0	3 distinct values, "string":1, "duh":1, "another string":1
Schema.b	Bool	3	0	1 false, 2 true
Schema.optional	Array	1	0
Schema.optional[]	Object	2	0	2 fields	string, maybe
Schema.optional[].string	String	2	0	2 distinct values, "s2":1, "s":1
Schema.optional[].maybe	Bool	1	0	0 false, 1 true
